### ISSUE TITLE
Quest: Cleansing an Empire <next to final step>

### ISSUE THREAD

**Issue by Rantric** [WarEmu/WarBugs#15441]
I am to search the Guilty Pyre, Rack and Cage. They are not to be found any where. I have approached all cages, racks and such in the highlighted area only to find none can be clicked. After reviewing videos of others walk throughs, the cage I've seen should be near the hero? I checked the other bug reports and only saw one related to this and it was the items were seen but blocked by a barrier. Thanks!

![Sulferbeard_001](https://user-images.githubusercontent.com/56928787/71991081-94715480-3202-11ea-97c1-6a9ab6f7914c.jpg)
![Sulferbeard_002](https://user-images.githubusercontent.com/56928787/71991082-94715480-3202-11ea-9c04-c7300d72789d.jpg)
![Sulferbeard_003](https://user-images.githubusercontent.com/56928787/71991083-94715480-3202-11ea-8729-4dab255cf4cd.jpg)
![Sulferbeard_004](https://user-images.githubusercontent.com/56928787/71991084-94715480-3202-11ea-9227-ac948c752d81.jpg)
![Sulferbeard_000](https://user-images.githubusercontent.com/56928787/71991086-94715480-3202-11ea-8f96-99ae798e7110.jpg)

**Comment by Ootootoo** [comment]
Can confirm that the step is impossible to complete.

**Comment by ngx83922** [comment]
When will the quest get fixed?

**Comment by Lheana** [comment]
When it will be said it is fixed.

**Comment by CrimsonFenris** [comment]
Experienced that too, today. Epic quests worked fine when I did them about a year ago, now even T3 dwarf is bugged (wrong final reward, see https://github.com/WarEmu/WarBugs/issues/15632)

**Comment by Confederatus** [comment]
It seems they amiss placed:
Guilty Pyre on east Unterbaum Graveyard (about 41400,40700)
Guilty Rack close to Cursed Mill (about 44040,49700)
Guilty Cage (not on Order riverside)

**Comment by Divrak88** [comment]
Can confirmed, still missing items as of 3/10. Figured I would post here rather than open another issue. Thank you for tracking this issue.

**Comment by Lheana** [comment]
There is no need to comment the issue to say "it's still bugged" when it's still opened and no one said it was fixed. We know it's not fixed cause no one said it was.

**Comment by Drayne69** [comment]
The quest is still broken.

**Comment by Lheana** [comment]
Fixed with next patch.
